src/main/java/nl/workingtalent/wtacademy/reservation/ReadReservationDto.java

```diff
@@ -0,0 +1,75 @@
+package nl.workingtalent.wtacademy.reservation;
+
+import java.time.LocalDate;
+
+import nl.workingtalent.wtacademy.loan.Loan;
+
+public class ReadReservationDto {
+
+	private long id;
+	private ReservationRequest reservationRequest;
+	private LocalDate requestDate;
+	private long book;
+	private long user;
+	private Long loan;
+
```

Comment:
Rename to bookId, userId en loanId. Duidelijker wat de variable precies contained.